src/modules/fancyzones/editor/FancyZonesEditor/Utils/FancyZonesEditorIO.cs

```diff
@@ -817,10 +817,45 @@ private bool SetCustomLayouts(List<CustomLayoutWrapper> customLayouts)
                     {
                         layout = ParseCanvasInfo(zoneSet);
                     }
+
+                    try
+                    {
+                        layout = new CanvasLayoutModel(zoneSet.Uuid, zoneSet.Name, LayoutType.Custom, zones, info.RefWidth, info.RefHeight);
+                    }
+                    catch (Exception)
+                    {
+                        continue;
+                    }
+
+                    layout.SensitivityRadius = info.SensitivityRadius;
+                }
```

Comment:
@niels9001 
this closing `}` is incorrect and it breaks the build.

Reply:
I opened a PR that should fix the build https://github.com/microsoft/PowerToys/pull/10585